lib/broccoli/ember-app.js

```diff
@@ -1845,9 +1852,28 @@ function addOutputFile(container, assetPath, options) {
     container[outputFile] = [];
   }
 
+  if (!allowImport(container[outputFile], assetPath, options)) { return; }
+
   if (options.prepend) {
     container[outputFile].unshift(assetPath);
   } else {
     container[outputFile].push(assetPath);
   }
 }
+
+function allowImport(fileList, assetPath, options) {
+  if (fileList.indexOf(assetPath) !== -1) {
+    logger.info('Highlander Rule: duplicate `app.import(' + assetPath + ')`. Only including the last by order.');
+
+    if (options.prepend) {
```

Comment:
This seems incorrect to me? The new import should be prepended and replace the existing one (because ordering matters 😜).